app/views/events/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= simple_form_for @event do |f| %>
```

Comment:
We need min_leaders, max_leaders, min_registrations, max_registrations as well.

Reply:
I have that stuff in my branch over here -> #29

Reply:
Added these in